Spatial.tscn

```diff
@@ -483,9 +484,9 @@ window_title = "Open a File"
 mode = 0
 access = 2
 filters = PoolStringArray( "*.png" )
-current_dir = "."
+current_dir = "/home/steph/Code/Projects/Burrito"
```

Comment:
I'm not sure let why this keeps changing without my knowledge.

Reply:
This is why I always do a `git diff <filename>` before `git add <filename>` and always do each file individually instead of `git add *`. You always want to know what you are adding.

This particular issue is a weird bug, idk if it is fixed yet or not but when you brows to a particular location it changes this path. The top-level `presubmit.sh` has 3 commands that work to remove all instances of it.